borg/remote.py

```diff
@@ -326,7 +335,7 @@ def handle_error(error, res):
                     self.unpacker.feed(data)
                     for unpacked in self.unpacker:
                         if not (isinstance(unpacked, tuple) and len(unpacked) == 4):
-                            raise Exception("Unexpected RPC data format.")
+                            raise Exception("Unexpected RPC data format while reading from server.")
```

Comment:
So what does your crystal ball say? Are there going to be hard to reproduce data format errors in the future?